packages/kcms/src/team/models/team.ts

```diff
@@ -62,7 +81,48 @@ export class Team {
     return this.clubName;
   }
 
-  public static new(arg: TeamCreateArgs): Team {
-    return new Team(arg.id, arg.teamName, arg.members, arg.isMultiWalk, arg.category, arg.clubName);
+  /**
+   * エントリー(当日参加するか)したかどうかを取得
+   */
+  getIsEntered(): boolean {
+    return this.isEntered;
+  }
+
+  /**
+   * エントリーする
+   * */
+  enter() {
+    this.isEntered = true;
+  }
+
+  /**
+   * エントリーを取り消す
+   * */
+  cancel() {
```

Comment:
何をキャンセルするのか分からないので、
```suggestion
  cancelEntry() {
```
などがいいかもしれません。